src/utils/formatters.ts

```diff
@@ -1,23 +1,18 @@
-import moment from 'moment-timezone';
+import moment from 'moment';
 import { PureObject } from '../interfaces';
 
 type TimeType = number | string | Date;
 
 export function formatDatetime(
   time: TimeType | undefined,
   defaultValue: string = '',
-  timezone: string = '',
   format: string = 'MMM D, YYYY HH:mm:ss'
 ): string {
-  return time
-    ? moment(time)
-        .tz(timezone || moment.tz.guess())
-        .format(format)
-    : defaultValue;
+  return time ? moment(time).format(format) : defaultValue;
```

Comment:
so it will be always utc now?

Reply:
In case of `moment(time)` call it will convert it to client local time